### PR TITLE
[PR] Person markup

### PR DESCRIPTION
**Pull request by philcable**
The related UC Projects output still needs to be customized - I'll tackle that in another PR.

The latest People Directory updates should also ship before this :)

### CODE REVIEW

File: wsu-people/person.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+					 alt="<?php echo esc_html( $display['name'] ); ?>" />
+			<?php } ?>
+
+			<?php if ( $display['link'] ) { ?></a><?php } ?>
+		</figure>
+		<?php } ?>
+
+		<hgroup>
+			<h1 class="article-title"><?php echo esc_html( $display['name'] ); ?></h1>
+			<span class="degree">MBA</span>
```
Comment (jeremyfelt) [inline wsu-people/person.php]: It looks like these didn't get replaced with dynamic content.

**Review by jeremyfelt — CHANGES_REQUESTED** [review]
I left one inline comment that should be addressed.

This looks good, but did get me thinking. Is there an opportunity to use something like `get_template_part( 'person', $display['directory_view'] )` to provide the different layouts either as part of `wsu-people/person.php` or elsewhere in the plugin before that template is loaded? This could give the plugin the opportunity to have some defaults that can be specifically overwritten.

File: wsu-people/person.php (hunk 1 above)
Comment (philcable) [reply wsu-people/person.php]: Good catch, I'll get that fixed!

**Comment by philcable** [comment]
I was thinking the same thing while building this out. Definitely a good idea, and I think it would be pretty straightforward to implement.

I also fibbed and added the University Center Projects output in this branch, so that bit will need a review too!

**Comment by philcable** [comment]
@jeremyfelt #reviewmerge?

The UC Projects output is still using the Content Syndicate markup, which might not be a big deal, but seems lazy to me and should probably change before launch.

**Review by jeremyfelt — APPROVED** [review]

**Comment by jeremyfelt** [comment]
It works as a start. We can see what it looks like and then adjust from there.

**Pull request merged**